### PR TITLE
Set better callable documentation for phpstan

### PR DESCRIPTION
**Pull request by gmponos**

### CODE REVIEW

File: src/Cookie/FileCookieJar.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ public function load(string $filename): void
 
         $data = Utils::jsonDecode($json, true);
         if (\is_array($data)) {
-            foreach (Utils::jsonDecode($json, true) as $cookie) {
+            foreach ($data as $cookie) {
```
Comment (gmponos) [inline src/Cookie/FileCookieJar.php:88]: @GrahamCampbell was there any reason decoding twice the data?

Comment (GrahamCampbell) [reply src/Cookie/FileCookieJar.php:88]: Don't remember. Did I write this???

Comment (gmponos) [reply src/Cookie/FileCookieJar.php:88]: Haven't checked history.. I guessed that you did because you worked on the `utils`.

Comment (gmponos) [reply src/Cookie/FileCookieJar.php:88]: Well you did and you didn't... https://github.com/guzzle/guzzle/pull/2546/files#diff-369479c0c2c035cfd85197bb3a8bfe6fR88

What seems to have happened is that you have only changed the functions.. the problem was there before too..

**Pull request merged**